docker/worker/start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "${TURBINIA_CONF}" | base64 -d > /etc/turbinia/turbinia.conf
```

Comment:
Should we add any error checking for non-existent environment variable and give an error message here?  Otherwise this will overwrite the config with an empty file if the var isn't populated.

Reply:
Agree. Have added a check if the environment variable exists (is not empty) and if the file exists (is not empty). This gives people the flexibility to eg mount a volume on /etc/turbinia and have a static configuration across all containers.